### DISCUSSION TITLE
new method

### DISCUSSION THREAD

**Discussion by CheesyPoofs3** [3kh0/ext-remover#110] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I found a new method to remove the extensions I might tell you how to do it

**Comment by 6829ASCS** [comment]
No you MIGHT not. You very much WILL.

Jokes aside please do share I'm desperate :)

**Comment by LogA1Cyber** [comment]
@CheesyPoofs3 what is your solution? b/c anything would be helpful

**Comment by Fallenmoon8080** [comment]
Does anyone else think he's just faking it to get clout, because I do? The guy can't even put a comma in his sentence. I highly doubt this is legitimate.

**Reply by synapt0** [reply]
Agreed

**Reply by CheesyPoofs3** [reply]
It's with dns

**Reply by LogA1Cyber** [reply]
school blocks DNS changes

**Reply by CheesyPoofs3** [reply]
Not my home wifi

**Reply by 6829ASCS** [reply]
Yeah I tried that method a few days ago and it doesn't work that well. DNS changes are also blocked at my school so its practically useless.

**Comment by 3kh0** [comment, marked as answer]
@CheesyPoofs3 Get blocked L

**Reply by The-Man-Is-Me** [reply]
🤣
